packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,106 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
```

Comment:
For some reason why I click on the https://deploy-preview-154--wonder-blocks.netlify.com/#!/Toolbar link it doesn't show the examples like it does for the other docs.

Reply:
Looks like styleguidist might only do this if the examples are associated with a doc in the components directory. I extracted the examples to `wonder-blocks-toolbar/components/toolbar.md` and they show up now.